src/main/java/org/folio/helper/CheckinReceivePiecesHelper.java

```diff
@@ -806,4 +807,14 @@ private static class PoLineAndTitleById {
     Map<String, CompositePoLine> poLineById;
     Map<String, Title> titleById;
   }
+
+  protected void addErrorForUpdatingItem(Piece piece, String error) {
+    if (error.contains(BARCODE_NOT_UNIQUE_MESSAGE)) {
```

Comment:
exceptions can be created without message, for example you can try just writing: new IllegalArgumentException(); and it will compile.
So it's required non-null check here, improving

Reply:
improved